website/src/css/components/_TestimonialCarousel.scss

```diff
@@ -0,0 +1,66 @@
+
+.testimonial-carousel {
+    .carousel-indicators {
```

Comment:
nit: the carousel indicator overlaps a bit on mobile:

![image](https://user-images.githubusercontent.com/16265452/69171961-ec28f480-0aca-11ea-9303-cd0e810f59ac.png)

Reply:
I've moved the carousel controls to the top to fix this.